src/module.ts

```diff
@@ -13,37 +12,25 @@ import {
   isNuxtMajorVersion,
 } from '@nuxt/kit'
 
-// @ts-expect-error no declaration file
-import defaultTailwindConfig from 'tailwindcss/stubs/config.simple.js'
-
 import { name, version, configKey, compatibility } from '../package.json'
 import * as resolvers from './resolvers'
 import logger, { LogLevels } from './logger'
 import { createExposeTemplates } from './expose'
 import { setupViewer, exportViewer } from './viewer'
-import { createInternalContext } from './context'
+import { createInternalContext } from './internal-context/load'
 
 import type { ModuleOptions, ModuleHooks } from './types'
 
 export type { ModuleOptions, ModuleHooks } from './types'
 
-const deprecationWarnings = (moduleOptions: ModuleOptions, nuxt = useNuxt()) =>
-  ([
-    ['addTwUtil', 'Use `editorSupport.autocompleteUtil` instead.'],
-    ['exposeLevel', 'Use `exposeConfig.level` instead.'],
-    ['injectPosition', `Use \`cssPath: [${
-      moduleOptions.cssPath === join(nuxt.options.dir.assets, 'css/tailwind.css')
-        ? '"~/assets/css/tailwind.css"'
-        : typeof moduleOptions.cssPath === 'string' ? `"${moduleOptions.cssPath}"` : moduleOptions.cssPath
-    }, { injectPosition: ${JSON.stringify(moduleOptions.injectPosition)} }]\` instead.`],
-  ] satisfies Array<[keyof ModuleOptions, string]>).forEach(
-    ([dOption, alternative]) => moduleOptions[dOption] !== undefined && logger.warn(`Deprecated \`${dOption}\`. ${alternative}`),
-  )
-
 const defaults = (nuxt = useNuxt()): ModuleOptions => ({
-  configPath: 'tailwind.config',
+  configPath: [],
   cssPath: join(nuxt.options.dir.assets, 'css/tailwind.css'),
-  config: defaultTailwindConfig,
+  config: {
```

Comment:
dynamic import tw stub when core supports async defaults (open a PR for it this week)